src/pages/feedbacks/FeedbackList.vue

```diff
@@ -3,7 +3,8 @@
   <main>
     <Wrapper>
       <BaseBar color="blue">
-        <ShowSuggestions />
+        //ShowSuggestions should be hidden in mobile - use Tailwind CC class
+        <ShowSuggestions :feedbacks="feedbacks" />
```

Comment:
you should do v-if here, because default is null.

Reply:
I updated my comment to remember your advice when I'll move to this step.